src/test/java/com/puppycrawl/tools/checkstyle/internal/XdocsPagesTest.java

```diff
@@ -1380,6 +1380,7 @@ else if (path.toFile().getName().contains("sun")) {
             }
 
             // these modules aren't documented, but are added to the config
+            styleChecks.remove("BeforeExecutionExclusionFileFilter");
```

Comment:
Isn't this the right way to do that?

Reply:
@mwkroening Yes it is. You can tell because you have no failing tests.
Something else is failing though:
> [checkstyle] [ERROR] /home/travis/build/checkstyle/checkstyle/src/main/resources/google_checks.xml:42: Line should not be longer than 100 symbols [RegexpSingleline]
[checkstyle] [ERROR] /home/travis/build/checkstyle/checkstyle/src/main/resources/google_checks.xml:83: Line should not be longer than 100 symbols [RegexpSingleline]

Reply:
But I didn't even touch these lines.
I just put code before them.

Reply:
Exactly. You code before them and so you changed the line numbers of the future text.
https://github.com/checkstyle/checkstyle/blob/4b8e5086a4994f18dff9e0cad874f922f1febd39/config/suppressions.xml#L8-L9

Reply:
Allright, I thought so. Thanks for showing me the right file to edit the suppressions!